### DISCUSSION TITLE
Busco equipo

### DISCUSSION THREAD

**Discussion by JVespid** [Launch-X-Latam/MisionFrontEnd#516] [Team Posts]
## Hola me llamo Bernardo Abel Lopez y soy  programador

Se usar html para maquetar y css para dar estilos al igual que tengo buena afinidad para javascript, también manejo las herramientas del editor de código al igual de ya saber utilizar git y GitHub tanto para trabajo en equipo como para despliegue final en un servidor 
- se un poco de react js
- se el lenguaje de Markdown para documentar proyectos 
_____
si alguien quisiera formal parte de mi equipo o quiera integrarme al suyo, pueden contactarme sin problema

**Comment by CAMILOITT** [comment]
hola mi nombre es Camilo estoy interesado en el grupo
mi correo: lx11002@innovaccion.mx

**Comment by Javilh97** [comment]
Hola soy Javier, Me gustaría formar parte de tu equipo si aun hay espacio :D, este es mi correo: lx15413@innovaccion.mx

**Comment by JosafatJimenezB** [comment]
Un compañero y yo buscamos equipo, sabemos un poco de VueJS, ReactJS, deploy, git y github, backend

**Comment by JVespid** [comment]
> Un compañero y yo buscamos equipo, sabemos un poco de VueJS, ReactJS, deploy, git y github, backend

me gustaría unirme a tu equipo

**Comment by Amhedriel** [comment]
## Buenas tardes JVespid
Escribo para postularme a tu equipo de ser posible, claro si aún hay vacantes, no soy un experto en prácticamente nada XD pero tengo algo de conocimientos en algunos campos, si piensas que puedo ser parte de la Alianza escríbeme a mi correo personal (que es con el que estoy registrado en GitHub para LaunchX y en donde están mis repositorios de las prácticas, aunque aún me falta un poco)

* davikin_01@hotmail.com

O a mi correo  de:

* lx06968@innovaccion.mx

---
Espero respuesta.

Saludos

**Comment by mafer23** [comment]
https://discord.gg/QZa2UAHu metanse aqui y no hacen spam
